frontend/src/components/DeliveryTime/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { formatAsHoursMinutes } from '../../helpers/time-helper';
+
+import './deliveryTime.scss';
+
+const DeliveryTime = props => {
+  const { time } = props;
+
+  function getTimeLabel(n) {
+    if (n < 60) {
+      return 'fast';
+    } else if (n < 120) {
+      return 'fair';
+    }
+    return 'slow';
+  }
+
+  return (
+    <p className={'delivery ' + getTimeLabel(time)}>
```

Comment:
Check out a package called [`classnames`](https://github.com/JedWatson/classnames).

Reply:
Great known package, definitely using it.